paasta_tools/cli/cmds/local_run.py

```diff
@@ -622,6 +622,14 @@ def get_local_run_environment_vars(instance_config, port0, framework):
         env["MARATHON_APP_LABELS"] = ""
         env["MARATHON_APP_ID"] = "/simulated_marathon_app_id"
         env["MARATHON_HOST"] = hostname
+        env["PAASTA_HOST"] = hostname
```

Comment:
I think this is the wrong place for these. The only reason we have these special variables is because they *come* from mararathon, so we have to simulate them.

The other variables are set further up the top.

But instead can you make this a function of the base instance class? That way they will automatically get applied, whether they happen in local run, or k8s, or spark, or tron, or whatever.

Reply:
I moved everything other than PAASTA_HOST to get_env. PAASTA_HOST is special so it's added sperately for k8s in kubernetes_tools, mesos in docker_wrapper, and in local_run.py.